.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+# Issue ticket number and link
+
+<https://github.com/wevisdemo/parliament-watch/issues/YOUR_ISSUE_NUMBER>
```

Comment:
GitHub has an official way to link issues to the PR. And if PR is merged to `main` branch, the issue will be closed automatically. [See more detail here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

We want contributors to use this method. So my suggestion is

```md
## Related GitHub issues

Close #<tag-related-issue-number-here>
```

Reply:
Done, thanks for document.